src/Lachain.Core/Blockchain/Pool/TransactionPool.cs

```diff
@@ -122,9 +148,9 @@ public OperatingError Add(TransactionReceipt receipt, bool notify = true)
             if (_transactions.ContainsKey(receipt.Hash))
                 return OperatingError.AlreadyExists;
             /* verify transaction before adding */
-            if (!(GetNextNonceForAddress(receipt.Transaction.From) >= receipt.Transaction.Nonce && TxNonceValid(receipt)))
+            if (!(GetNextNonceForAddress(receipt.Transaction.From) == receipt.Transaction.Nonce))
```

Comment:
Does it mean that we can't add tx to pool if its nonce is bigger then next nonce value?